AssignmentJenkins/FreeStyle-DSL-Script/FreeStyle_DSL_Script.groovy

```diff
@@ -0,0 +1,12 @@
+job('DSL-Tutorial-1-Test') {
+    scm {
+        git('https://github.com/leereilly/hello-world-java.git')
```

Comment:
how you will use username and password for this repo?

Reply:
This repository is public. So there is no credential required.